buildSrc/src/main/kotlin/ComponentVersions.kt

```diff
@@ -5,7 +5,7 @@ object ComponentVersions {
     const val ratingBarVersion = "1.0.2"
     const val imageSliderVersion = "1.0.8"
     const val phoneNumberVersion = "1.0.2"
-    const val dialogsVersion = "1.2.9"
+    const val dialogsVersion = "1.3.0"
```

Comment:
Don't forget to update version info on README.md#4

Reply:
Done https://github.com/Trendyol/android-ui-components/pull/93/commits/46670bfd2e8113bfb2b4723bd0e71e49bffd893e